libr/util/sandbox.c

```diff
@@ -13,15 +13,16 @@
 
 static bool enabled = false;
 static bool disabled = false;
+static const char *wwwroot = NULL;
```

Comment:
dont use global vars, this stuff is unnecessary here

Reply:
#10564